app/src/main/java/com/superproductivity/superproductivity/FullscreenActivity.java

```diff
@@ -109,7 +109,7 @@ public boolean shouldOverrideUrlLoading(WebView view, String url) {
 
             boolean IS_DEBUG = 0 != (getApplicationInfo().flags & ApplicationInfo.FLAG_DEBUGGABLE);
 
-            jsi = new JavaScriptInterface(this, wv, IS_DEBUG);
```

Comment:
I've switched variants in Android Studio and now seen that this is required for the `play` dimension. Will fix.

Reply:
Leaving this back in renders F-Droid variants unbuildable.

Reply:
Hey tere! Thank you very much for opening this up. What's the state on this? As the web token is the same both for production and for dev, I think we can remove passing IS_DEBUG from the JavaScriptInterface completely. So I would either remove it completely or remove this change.

Reply:
No worries!